main.go

```diff
@@ -608,6 +613,12 @@ func statusAction(ctx *cli.Context) (err error) {
 		fmt.Printf("\nManage your connected systems: https://red.ht/connector\n")
 	}
 
+	// At the end check if all statuses are correct.
+	// If not, return 1 exit code without any message.
+	if systemStatus.returnCode != 0 {
+		return cli.Exit("", 1)
```

Comment:
Shouldn't this be?

```suggestion
		return cli.Exit("", systemStatus.returnCode)
```

Reply:
Well, the value of `systemStatus.returnCode` depends on "failed" sub-statuses. It could be 1, 2, 3, but we return value 1 for all other error cases.

I think that we should return different values for various situations (not only 1), but we should think twice before we do it and create some design document for that.

Reply:
Ok, so this change is just applying a non-zero return code when any sub-status is non-zero. Seems reasonable.